tox.ini

```diff
@@ -69,6 +71,9 @@ setenv =
     async: PYTEST_ADDOPTS = --async_only
     async: PYTEST_PATH = napari
 deps =
+    pytest-cov
+    pyqt6: PyQt6
+    pyside6: PySide6
```

Comment:
why this additional deps are required?

Reply:
those will only get used when the respective factors hit (i.e. when `backend` in the action is the corresponding value).  The alternative is to put them as extras directly in setup.cfg... but I was leaving that for later, when we actually felt like we wanted to "support" them